### PR TITLE
chore: update Service module

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

- [x] added Service description;
- [x] update Service Creator's fields

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/pages/Service/components/Step1.tsx [hunk 1]
```diff
@@ -45,11 +45,11 @@ const Step1: React.FC<ServiceModule.Step1PassProps> = ({
   return (
     <>
       <Form {...FORM_LAYOUT} form={form}>
-        <Form.Item name="name" label={formatMessage({ id: 'component.global.name' })}>
-          <Input disabled={disabled} />
+        <Form.Item name="name" label={formatMessage({ id: 'component.global.name' })} required>
```
Comment (liuxiran) [inline web/src/pages/Service/components/Step1.tsx:48]:
```suggestion
        <Form.Item name="name" label={formatMessage({ id: 'component.global.name' })} rules={[{ required: true, message: 'Please input your service name!' }]}>
```

Comment (juzhiyuan) [reply web/src/pages/Service/components/Step1.tsx:48]: good, let me commit.

**Review by membphis — APPROVED** [review]

**Comment by juzhiyuan** [comment]
ping @LiteSun @bzp2010

**Review by LiteSun — APPROVED** [review]

**Review by starsz — APPROVED** [review]
Description LGTM

**Review by nic-chen — APPROVED** [review]

**Pull request merged**